bool motor_on
---
bool was_set
